pkg/crc/machine/machine.go

```diff
@@ -191,6 +185,9 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		logging.InfoF(" To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=%s'", result.ClusterConfig.KubeConfig)
 		logging.InfoF(" Access the OpenShift web-console here: %s", result.ClusterConfig.ClusterAPI)
 		logging.InfoF(" Login to the console with user: kubeadmin, password: %s", result.ClusterConfig.KubeAdminPass)
+		if os.CurrentOS() == os.DARWIN {
+			logging.WarnF(fmt.Sprintf(" Make sure add 'nameserver %s' as first entry to '/etc/resolv.conf' file", instanceIP))
```

Comment:
this should be in the`runPostStartForOS` as this is specific to the hypervisor.

Can be a message for now... but needs to be automated.